tests/gordo_components/data_provider/test_data_provider_iroc.py

```diff
@@ -105,6 +122,32 @@ def test_load_series_no_tag_list(self):
         )
         self.assertEqual([], res)
 
+    def test_load_series_many_assets(self, _mocked_method):
```

Comment:
The _mocked_method require the use of mocking fixtures. I don't think you need it here, however, as these tests should cause an early return, before the reading of files are started.